### PR TITLE
Implement audio increase/decrease

### PR DESCRIPTION
**Pull request by MrTanoshii**
### Feature

- Increase audio volume with `+`, `=`
- Decrease audio volume with `-`

### Misc

- Add documentation

## Checklist

- [x] My code follows the code style of this project [How to use `black` autoformatter](https://github.com/psf/black#usage).
- [x] I have applied relevant labels and linked the PR to the relevant Issue.

### CODE REVIEW

File: src/audio/audio.py [hunk 1]
```diff
@@ -9,27 +9,30 @@
 class Audio:
     """Audio class."""
 
-    sound_list = {}
+    sound_dict = {}
```
Comment (mihett05) [inline src/audio/audio.py:12]: :=

File: src/views/map.py [hunk 2]
```diff
@@ -201,6 +203,16 @@ def on_key_press(self, symbol, _modifiers):
                 self.bgm_player = Audio.play_random(["bgm_1", "bgm_2"])
         elif symbol == arcade.key.S:
             self.enemy_handler.send_wave(*C.Waves.wave_1_1())  # TODO: change this
+        # Reduce master volume | -
+        elif symbol == arcade.key.MINUS or symbol == arcade.key.NUM_SUBTRACT:
```
Comment (mihett05) [inline src/views/map.py:298]: :=

**Review by mihett05 — DISMISSED** [review]
:=

**Review by Jeb4dev — DISMISSED** [review]
**:=**

**Review by Jeb4dev — APPROVED** [review]

**Review by mihett05 — APPROVED** [review]
:=

**Pull request merged**